npm/private/utils.bzl

```diff
@@ -156,6 +182,113 @@ def _npm_registry_download_url(package, version, registries, default_registry):
 def _is_git_repository_url(url):
     return url.startswith("git+ssh://")
 
+def _is_url(url):
+    return url.find("://") != -1
```

Comment:
How about `is_probably_url`?

Reply:
Right. That function is not very robust. I'll just move it back inline to the one file that needs it.